major: start a big renovation (#22)
Alzymologist/alzymologist.github.io: content/info/hardware.md

+++
title = "Hardware development"
date = 2024-01-13
+++

> ## Contract hardware development

Reviewer: maybe instead:
Hardware development and consulting

Author: ```suggestion
## Hardware development and consulting
```

> We offer contract hardware development of high complexity. Expertise includes but is not limited to:

Reviewer: and then

> We offer contract hardware development of high complexity.

We offer contract hardware development and consulting services for complex projects.

Author: why not use suggestions?

```suggestion
We offer contract hardware development and consulting services for complex projects. Expertise includes but is not limited to:
```

Reviewer: noted, never tried it before, will try in the future
